CHANGELOG.md

```diff
@@ -29,13 +29,20 @@
     - ⚠️ `enableCellSelect`
     - ⚠️ `getValidFilterValues`
     - ⚠️ `onCellCopyPaste`
-    - ⚠️ `onGridKeyDown`
-    - ⚠️ `onGridKeyUp`
-    - ⚠️ `onRowDoubleClick`
     - ⚠️ `rowsContainer`
+    - ⚠️ `onRowDoubleClick`, `onRowClick`, `onGridKeyDown`, `onGridKeyUp`
+      - Use standard `div` props on the `Row` component itself instead, via `rowRenderer`:
+      ```jsx
+      function RowRenderer(props) {
+        return <Row onClick={(event) => {}} {...props} />;
```

Comment:
if the new way is to use `onClick` we should pass the old props to the clickEvent as well or as a currying function. so we can identify quickly which cell has been clicked on. Otherwise we have to do some DOM querySelector.

For instance, I want to navigate when pressing ctrl+click on the row, so I look for event.ctrlKey === true sent by the user and a click performed anywhere but on column.key === 'select-row'. currently there was a way with onRowClick to do with and using `window.event` to capture the current event.